app/controllers/notifications_controller.rb

```diff
@@ -66,4 +67,12 @@ def if_not_signed_in
         end
       end
     end
+
+    # Stops here if the signed in user is not the owner of
+    # the notification set by :set_notification.
+    def ensure_user_owns_notification!
+      unless @notification.user == current_user
```

Comment:
@chrisrp So that signed_in method exists for `notifications.js` (https://github.com/julianguyen/ifme/blob/bd3a6ab6da7a7c1b281bff20c13d0970e8c54362/app/assets/javascripts/notifications.js#L197).